### ISSUE TITLE
Missing CancellationToken

### ISSUE THREAD

**Issue by nerumo** [ericnewton76/gmaps-api-net#129]
I haven't checked many Api calls but at least some are missing the CancellationToken on the async Apis.

**Comment by ericnewton76** [comment]
Indeed.  We had a first phase of adding async support and probably need to implement cancellation semantics.
